.gitignore

```diff
@@ -2,3 +2,6 @@
 composer.lock
 /phpunit.xml
 .phpunit.result.cache
+/.idea
+/.vscode
+/.rnd
```

Comment:
I'd prefer if you removed these and added these to a global gitignore file: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files